### DISCUSSION TITLE
Janet 1.17.1

### DISCUSSION THREAD

**Discussion by bakpakin** [janet-lang/janet#776] [General]
Small patch release that brings back the MacOS and Windows builds, as well as fixing a few bugs.

<hr /><em>This discussion was created from the release <a href='https://github.com/janet-lang/janet/releases/tag/v1.17.1'>Janet 1.17.1</a>.</em>

**Comment by sogaiu** [comment]
Thanks for the release!

I had success building and installing from e1c4fc29debeee95f1207cb4791f659e6bd70f7f on a Linux box and a Windows VM.

I also tried the new [`install-jpm-git` Makefile target](https://github.com/janet-lang/janet/blob/e1c4fc29debeee95f1207cb4791f659e6bd70f7f/Makefile#L284-L293) on Linux.  It seems to have worked.  Subsequent to installing jpm I tried using it to build and install a project.

Update: inspired by pyrmont, I tried the x64 installer for Windows.  That seemed to work fine too.

**Comment by pyrmont** [comment]
Reporting that I had success using the pre-built binaries for Linux and macOS :)
